builtin/logical/pki/path_tidy.go

```diff
@@ -23,15 +23,16 @@ import (
 
 var tidyCancelledError = errors.New("tidy operation cancelled")
 
+//go:generate enumer -type=tidyStatusState -trimprefix=tidyStatus
 type tidyStatusState int
 
 const (
-	tidyStatusInactive   tidyStatusState = iota
```

Comment:
I guess my other question is whether it's necessary to add full enum functionality to those that were simply used with iota.  Adds code that isn't used...